templates/accordion.hbs

```diff
@@ -1,32 +1,59 @@
-<div class="accordion-inner component-inner">
-    {{> component this}}
-    <div class="accordion-widget component-widget">
-        {{#each _items}}
-        <div class="accordion-item{{#if _classes}} {{_classes}}{{/if}}" data-index="{{_index}}">
-            <button id="{{../_id}}-{{@index}}-accordion-button" class="base accordion-item-title {{#if _isVisited}}visited{{/if}}" aria-expanded="false">
-                <div class="accordion-item-title-inner">
-                    <div class="accordion-item-title-icon icon icon-plus h5" aria-hidden="true"></div>
-                    {{{title}}}
-                </div>
-            </button>
-            <div role="region" class="accordion-item-body" aria-labelledby="{{../_id}}-{{@index}}-accordion-button">
-                <div class="accordion-item-body-inner">
-                    <div class="accordion-item-text">
-                        {{{compile body}}}
-                    </div>
-                    {{#if _graphic}}
-                    {{#if _graphic.src}}
-                    <div class="accordion-item-graphic">
-                        <img src="{{_graphic.src}}" aria-label="{{_graphic.alt}}">
-                    </div>
-                    {{#if _graphic.attribution}}
-                        <div class="graphic-attribution">{{{_graphic.attribution}}}</div>
-                    {{/if}}
-                    {{/if}}
-                    {{/if}}
-                </div>
+<div class="component__inner accordion__inner">
+
+  {{> component this}}
+
+  <div class="component__widget accordion__widget">
+
+    {{#each _items}}
+    <div class="accordion__item{{#if _graphic.src}} has-image{{/if}}{{#if _classes}} {{_classes}}{{/if}}" data-index="{{_index}}">
+
+      <button id="{{../_id}}-{{@index}}" class="accordion__item-btn js-toggle-item is-closed{{#if _isVisited}} is-visited{{/if}}" aria-expanded="false">
+        <div class="accordion__item-btn-inner">
+
+          <div class="accordion__item-icon">
+            <div class="icon"></div>
+          </div>
+
+          <div class="accordion__item-title">
+            <div class="accordion__item-title-inner">
+              {{{title}}}
             </div>
+          </div>
+
         </div>
-        {{/each}}
+      </button>
+
+      <div class="accordion__item-content" role="region" aria-labelledby="{{../_id}}-{{@index}}-accordion-button">
```

Comment:
Is this correct
`aria-labelledby="{{../_id}}-{{@index}}-accordion-button"`

or should it be
`aria-labelledby="{{../_id}}-{{@index}}"`?

Reply:
The button should probably be updated to add -accordion-button, to make the id more specific, descriptive and unique. Has a convention been established for item ids in v5?